core/aws_ddk_core/resources/_utils.py

```diff
@@ -72,6 +72,11 @@ def pandas_sdk_layer(
     region: Optional[str] = "us-east-1",
     version: Optional[str] = None,
 ) -> lmbda.LayerVersion:
+
+    context_layer = scope.node.try_get_context("pandas_sdk_lambda_layer_version_arn")
```

Comment:
Have you tested that this reads data from `cdk.context.json` (or `ddk.context.json`)?

Also, do you think we should also be using [construct.node.setContext()](https://docs.aws.amazon.com/cdk/api/v1/docs/@aws-cdk_core.ConstructNode.html#setwbrcontextkey-value)? This way, after we've scanned for the layer version the first time, we'll automatically store it in the context. Then, when customers want to upgrade, they just delete the value from the context, and CDK synth will discover the newest one.

Reply:
I've only tested from `cdk.json` so far, I'm in favor of setting layer arn pulled during synth as context. I'll put this in and make sure to add this all to docs as well.

Reply:
So for now I've left out set context as it doesn't look possible but this can be pulled from cdk context and `ddk.json`